examples/prostate/configs/prostate_ditto/config/environment.json

```diff
@@ -0,0 +1,3 @@
+{
+    "APP_CKPT_DIR": "models"
+}
```

Comment:
What's the difference between config_train.json and environment.json?

Can we merge this two files into one?

Reply:
environment.json is needed by nvflare some versions ago, seems no longer the case, will remove it